.travis.yml

```diff
@@ -15,7 +15,7 @@ before_install:
 before_script:
   - psql -c 'create database high_performance_java_persistence;' -U postgres
 script:
-  - mvn verify
+  - mvn verify -Ptravis
```

Comment:
Could you split this into two Pull Requests? I'd like to have the original one separated from the Travis one. You should create a separate issue for Travis.

Thanks.

Reply:
sure